tests/client/button-count.test.js

```diff
@@ -0,0 +1,82 @@
+import { JSDOM } from 'jsdom';
+//import { jest } from '@jest/globals';
+
+import { mockConnection } from '../../client/js/connection.js';
+import { addWidget } from '../../client/js/serverstate.js';
+import { Button } from '../../client/js/widgets/button.js';
+import { Label } from '../../client/js/widgets/label.js';
+
+const loadWindow = async () => {
```

Comment:
I'm quite new to async/await/Promises but I'm pretty sure this is too complicated. Doesn't `loadWindow = new Promise(...)` work?

Reply:
No, but `async function loadWindow() { ... ` does.

Reply:
I tried getting rid of the `loadWindow()` function, but couldn't get it to work with simpler code than simply declaring it `async`.